File: BE/HouseKepperConnect/BusinessObject/DTO/RatingDisplayDTO.cs
using System.ComponentModel.DataAnnotations;

namespace BusinessObject.DTO
{
    public class RatingDisplayDTO
    {
        public int RatingID { get; set; }

        public int FamilyID { get; set; }

        public int HouseKeeperID { get; set; }
        public string FamilyName { get; set; }

        [Required]
        [MaxLength(500)]
        public string Content { get; set; }

        [Range(1, 5)]
        public int Score { get; set; }

        [Required]
        public DateTime CreateAt { get; set; }
    }
}

File: Mobile/app/src/main/java/com/example/housekeeperapplication/Model/DTOs/RatingDisplayDTO.java
package com.example.housekeeperapplication.Model.DTOs;

public class RatingDisplayDTO {
    private int ratingID;
    private int familyID;
    private int houseKeeperID;
    private String content;
    private int score;
    private String createAt;

    public int getRatingID() {
        return ratingID;
    }

    public void setRatingID(int ratingID) {
        this.ratingID = ratingID;
    }

    public int getFamilyID() {
        return familyID;
    }

    public void setFamilyID(int familyID) {
        this.familyID = familyID;
    }

    public int getHouseKeeperID() {
        return houseKeeperID;
    }

    public void setHouseKeeperID(int houseKeeperID) {
        this.houseKeeperID = houseKeeperID;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }

    public String getCreateAt() {
        return createAt;
    }

    public void setCreateAt(String createAt) {
        this.createAt = createAt;
    }
}
